### PR TITLE
Update. All. Replace brands data to apbct->data[wl_*].

### PR DESCRIPTION
**Pull request by svfcode**

### CODE REVIEW

**Review by ArtemAnoshin — APPROVED** [review]

File: inc/cleantalk-settings.php [hunk 1]
```diff
@@ -1155,20 +1165,20 @@ function apbct_settings__display()
...
              . '</a><br/>';
-        echo __('Use s@cleantalk.org to test plugin in any WordPress form.', 'cleantalk-spam-protect') . '<br>';
-        echo __('CleanTalk is registered Trademark. All rights reserved.', 'cleantalk-spam-protect') . '<br/>';
+        echo __('Use stop_email@example.com to test plugin in any WordPress form.', 'cleantalk-spam-protect') . '<br>';
+        echo $apbct->data['wl_brandname_short'] . __(' is registered Trademark. All rights reserved.', 'cleantalk-spam-protect') . '<br/>';
         if ( $apbct->key_is_ok ) {
             echo '<b style="display: inline-block; margin-top: 10px;">' . sprintf(
-                __('Do you like CleanTalk? %sPost your feedback here%s.', 'cleantalk-spam-protect'),
+                __('Do you like ' . $apbct->data['wl_brandname_short'] . '? %sPost your feedback here%s.', 'cleantalk-spam-protect'),
                 '<a href="https://wordpress.org/support/plugin/cleantalk-spam-protect/reviews/#new-post" target="_blank">',
```
Comment (alexandergull) [inline inc/cleantalk-settings.php:1182]: Does WL dashboard logic support feedback link replacement? If so, we should replace it too, if not, just remove this block.

File: lib/Cleantalk/ApbctWP/AdminNotices.php [hunk 2]
```diff
@@ -192,6 +194,8 @@ public function notice_trial() // phpcs:ignore PSR1.Methods.CamelCapsMethodName.
      */
     public function notice_renew() // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
     {
+        global $apbct;
+
         if ( $this->apbct->notice_show && $this->apbct->notice_renew == 1 && $this->apbct->moderate_ip == 0 && ! $this->apbct->white_label ) {
             $renew_link  = "<a href=\"https://cleantalk.org/my/bill/recharge?utm_source=wp-backend&utm_medium=cpc&utm_campaign=WP%%20backend%%20renew$this->user_token&cp_mode=antispam\" target=\"_blank\">%s</a>";
```
Comment (alexandergull) [inline lib/Cleantalk/ApbctWP/AdminNotices.php:200]: Probably we should replace this with mailto link if it was provided by wl.

**Review by alexandergull — CHANGES_REQUESTED** [review]

File: inc/cleantalk-settings.php (hunk 1 above)
Comment (Glomberg) [reply inc/cleantalk-settings.php:1182]: +1

**Review by Glomberg — CHANGES_REQUESTED** [review]
Please remove hyperlinks if the WL mode was enabled.

File: inc/cleantalk-settings.php (hunk 1 above)
Comment (svfcode) [reply inc/cleantalk-settings.php:1182]: There already checked $apbct->white_label mode, on 1155 line.

File: lib/Cleantalk/ApbctWP/AdminNotices.php (hunk 2 above)
Comment (svfcode) [reply lib/Cleantalk/ApbctWP/AdminNotices.php:200]: One of condition for show notice on client dashboard $this->apbct->white_label. It's mean notice already hidden in white_label mode.

**Review by alexandergull — APPROVED** [review]

**Pull request merged**